keyboards/cannonkeys/satisfaction75/keymaps/boy_314/config.h

```diff
@@ -0,0 +1 @@
+#define ENCODER_RESOLUTION 2
```

Comment:
You want to add the include guard, to prevent issues.

```suggestion
#pragma once 

#define ENCODER_RESOLUTION 2
```